modules/images/webp-uploads/fallback.js

```diff
@@ -0,0 +1,75 @@
+window.wpPerfLab = window.wpPerfLab || {};
+
+( function( document ) {
+	window.wpPerfLab.webpUploadsFallbackWebpImages = function( media ) {
+		for ( var i = 0; i < media.length; i++ ) {
+			try {
+				var ext = media[i].media_details.sources['image/jpeg'].file.match( /\.\w+$/i );
```

Comment:
Let's be a bit more defensive here. There could be images that don't have the `image/jpeg` MIME type even available, or some images may not even have `sources` set - both cases would result in an error here right now.
So let's cater for that, e.g. like this:
```suggestion
				if ( ! media[i].media_details.sources || ! media[i].media_details.sources['image/jpeg'] ) {
					continue;
				}

				var ext = media[i].media_details.sources['image/jpeg'].file.match( /\.\w+$/i );
```